### PR TITLE
Release v7.0.0-alpha.2

### PR DESCRIPTION
**Pull request by LukasTy**
### Release the packages

- [ ] Checkout the last version of the working branch
- [ ] Make sure you have the latest dependencies installed: `yarn`.
- [ ] Build the packages: `yarn release:build`.
- [ ] Release the versions on npm: `yarn release:publish` (you need your 2FA device).
- [ ] Push the newly created tag: `yarn release:tag`.

### Publish the documentation

The documentation must be updated on the `docs-vX` branch (`docs-v4` for `v4.X` releases, `docs-v5` for `v5.X` releases, ...)

- [ ] Push the working branch on the documentation release branch to deploy the documentation with the latest changes.

```bash
git push upstream next:docs-next -f
```

You can follow the deployment process [on the Netlify Dashboard](https://app.netlify.com/sites/material-ui-x/deploys?filter=docs-next)
Once deployed, it will be accessible at https://material-ui-x.netlify.app/ for the `docs-next` deployment.

### Publish GitHub release

- [ ] After documentation is deployed, publish a new release on [GitHub releases page](https://github.com/mui/mui-x/releases)

### CODE REVIEW

**Review by michelengelen — APPROVED** [review]
Nice!!!

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,124 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.0.0-alpha.2
+
+_Nov 23, 2023_
+
+We'd like to offer a big thanks to the 10 contributors who made this release possible. Here are some highlights ✨:
+
+- 📈 Export missing Charts props
```
Comment (LukasTy) [inline CHANGELOG.md:12]: Do you see anything else worth highlighting?

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,124 @@
...
+
+- The `adapter.dateWithTimezone` method has been removed and its content moved to the `date` method.
+  You can use the `adapter.date` method instead:
+
+  ```diff
+  - adapter.dateWithTimezone(undefined, 'system');
+  + adapter.date(undefined, 'system');
+  ```
+
+- The `adapter.date` method now has the v6 `adapter.dateWithTimezone` method behavior.
```
Comment (flaviendelangle) [inline CHANGELOG.md:59]: I forgot one breaking change:

```suggestion
- The `dayPickerClasses` variable has been renamed `dayCalendarClasses` to be consistent with the new name of the `DayCalendar` component introduced in v6.0.0.

  ```diff
  - import { dayPickerClasses } from '@mui/x-date-pickers/DateCalendar';
  + import { dayCalendarClasses } from '@mui/x-date-pickers/DateCalendar';
```

- The `adapter.date` method now has the v6 `adapter.dateWithTimezone` method behavior.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,124 @@
...
+- [docs] Fix typos and grammar issues (#11049) @flaviendelangle
+- [docs] Fix version links (#11001) @LukasTy
+- [docs] Point to the source of `@mui/x-data-grid-generator` (#11134) @oliviertassinari
+
+### Core
+
+- [core] Bump monorepo (#11160) @LukasTy
+- [core] Fix comment in doc generation (#11098) @flaviendelangle
+- [core] Rename OpenCollective @oliviertassinari
+- [core] Upgrade babel-plugin-module-resolver to 5.0.0 (#11065) @flaviendelangle
```
Comment (flaviendelangle) [inline CHANGELOG.md:119]:
```suggestion
- [core] Upgrade `babel-plugin-module-resolver` to 5.0.0 (#11065) @flaviendelangle
```

File: CHANGELOG.md (hunk 1 above)
Comment (flaviendelangle) [reply CHANGELOG.md:12]: Nothing on pickers / tree view

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,124 @@
...
+  + adapter.date('2022-04-17');
+
+  - adapter.date(new Date(2022, 3, 17, 4, 5, 34));
+  + adapter.date('2022-04-17T04:05:34');
+
+  - adapter.date(new Date('Invalid Date'));
+  + adapter.getInvalidDate();
+  ```
+
+- The deprecated `defaultCalendarMonth` prop has been removed in favor of the more flexible `referenceDate` prop.
```
[the remark this answers is not in the document]
Comment (LukasTy) [reply CHANGELOG.md]: Done. 👌

**Review by flaviendelangle — APPROVED** [review]
Let's go!

**Pull request merged**